### PR TITLE
S3/Redshift Support

### PR DESCRIPTION
**Pull request by iter-io**
Please let me know if including any of this code in the repo might be of interest.  In that case I can merge the latest changes into this branch and write some documentation.

### CODE REVIEW

File: dags/load_dag_redshift.py [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+def load_task(ds, **kwargs):
+    conn_id = kwargs.get('conn_id')
+    file_format = kwargs.get('file_format')
+    task = kwargs.get('task')
+    pg_hook = PostgresHook(conn_id)
+
+    if file_format == 'csv':
+        sql = """
+            COPY {schema}.{table}
```
Comment (medvedev1088) [inline dags/load_dag_redshift.py]: To make this operation idempotent you may need to delete the records for the current `ds` before loading the file.

File: dags/load_dag_redshift.py [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+        output_bucket=output_bucket,
+        date=ds,
+        file_format=file_format,
+        aws_access_key_id=aws_access_key_id,
+        aws_secret_access_key=aws_secret_access_key
+    )
+    pg_hook.run(formatted_sql)
+
+
+def add_load_tasks(task, file_format, allow_quoted_newlines=False):
```
Comment (medvedev1088) [inline dags/load_dag_redshift.py]: Seems `allow_quoted_newlines` is not used.

File: dags/load_dag_redshift.py (hunk 1 above)
Comment (iter-io) [reply dags/load_dag_redshift.py]: @medvedev1088 - Addressed by [bbfb7684f37337363db0894a7b8e65c6488f4a22](https://github.com/iter-io/ethereum-etl-airflow/commit/bbfb7684f37337363db0894a7b8e65c6488f4a22)

File: dags/load_dag_redshift.py (hunk 2 above)
Comment (iter-io) [reply dags/load_dag_redshift.py]: @medvedev1088 - Addressed by [bbfb7684f37337363db0894a7b8e65c6488f4a22](https://github.com/iter-io/ethereum-etl-airflow/commit/bbfb7684f37337363db0894a7b8e65c6488f4a22)

**Comment by iter-io** [comment]
@medvedev1088 - I addressed your comments above.  I also updated the export DAGs to use `build_export_dag.py` and created a new helper called build_load_dag_redshift.

File: dags/variables.py [hunk 3]
```diff
@@ -62,6 +66,23 @@ def read_load_dag_vars(var_prefix, **kwargs):
...
+        'aws_access_key_id':  read_var('aws_access_key_id', var_prefix, True, **kwargs),
+        'aws_secret_access_key': read_var('aws_secret_access_key', var_prefix, True, **kwargs),
+        'notification_emails': read_var('notification_emails', None, False, **kwargs),
+        'schedule_interval': read_var('schedule_interval', var_prefix, True, **kwargs),
+    }
+
+    load_start_date = read_var('load_start_date', var_prefix, False, **kwargs)
+    if load_start_date is not None:
+        load_start_date = datetime.strptime(load_start_date, '%Y-%m-%d')
+        var_prefix['load_start_date'] = load_start_date
```
Comment (medvedev1088) [inline dags/variables.py]: I had a bug in my code. It should be `vars` instead of `var_prefix` here.

Comment (iter-io) [reply dags/variables.py]: @medvedev1088 - This is what you were referring to, right?

https://github.com/iter-io/ethereum-etl-airflow/commit/54120f414a6562b7a81be06e5ed112140f3c4134

**Comment by iter-io** [comment]
@medvedev1088 - I'm going to thoroughly test the most recent commit tonight on my AWS setup.  But I won't be able to test the google functionality.

**Comment by iter-io** [comment]
@medvedev1088 - I'm getting this error now:

`ModuleNotFoundError: No module named 'google.storage'`

Not sure how to install all dependencies for this project.  I think I originally figured it out via trial and error.  I expect other people will get errors from my code because of not having the boto3 module installed.

**Pull request merged**